packages/editor-ui/src/components/NodeCreateList.vue

```diff
@@ -41,7 +41,7 @@ export default Vue.extend({
 	},
 	computed: {
 		nodeTypes (): INodeTypeDescription[] {
-			return this.$store.getters.allNodeTypesDefaultVersion;
+			return this.$store.getters.allNodeTypes.filter((x: INodeTypeDescription) => !x.defaultVersion || (x.defaultVersion === x.version));
```

Comment:
it's clearer if use a descriptive parameter
```suggestion
			return this.$store.getters.allNodeTypes.filter((type: INodeTypeDescription) => !type.defaultVersion || (type.defaultVersion === x.version));
```